Assets/_Main/Scripts/Assistance/NewScriptStartup.cs

```diff
@@ -0,0 +1,35 @@
+/// <summary>
+/// --New script startup--<para></para>
+/// 
+/// Script used to be copied and add my most common regions, those can be deleted after if unused
+/// </summary>
+/// 
+public class NewScriptStartup
+{
+    #region FIELDS
+
+
+
+    #endregion
+
+    #region PROPERTIES
+
+
+
+    #endregion
+
+    #region METHODS
+
+    /// <summary> Summary of the function. </summary>
+    /// <remarks> Add some remarks if needed. </remarks>
+    /// <returns>Returns <paramref name="a"/> +2 if used :V</returns>
+    /// 
+    private int DoNothing(int a){ return a + 2; }
+    #endregion
+
+    #region MONOBEHAVIOUR
+
+
+
+    #endregion
+}
```

Comment:
Remove this script, you can change the new scripts templates on the Unity installation folder, look for folder ´ScriptTemplates´